### PR TITLE
Task/hybrid loss

### PR DESCRIPTION
**Pull request by M1croZavr**
Ongoing progress on hybrid loss task...

### CODE REVIEW

File: scripts/activation_distillation.py [hunk 1]
```diff
@@ -1,20 +1,30 @@
...
+
+import torch
 import transformers
 from datasets import load_dataset
+from transformers import AutoModelForCausalLM, AutoTokenizer, DataCollatorForLanguageModeling
 
-from train.arguments import MyTrainingArguments
-from train.trainer import MyTrainer
+from compression_horizon.train.arguments import MyTrainingArguments
+from compression_horizon.train.trainer import MyTrainer
```
Comment (mrsndmn) [inline scripts/activation_distillation.py]: такие изменения не очень ок, потому что куча конфликтов сейчас будет, но идею понял, ты прав, так будет красивее и лучше

Comment (mrsndmn) [reply scripts/activation_distillation.py]: возьмем твою версию

File: src/compression_horizon/train/trainer.py [hunk 2]
```diff
@@ -29,84 +30,116 @@ def __init__(
...
                 elif loss_type == "cosine":
-                    cos = F.cosine_similarity(tgt, pred, dim=-1)
-                    loss = loss + (1.0 - cos).mean()
+                    cosine = F.cosine_similarity(compression_hidden_states, target_hidden_states, dim=-1)
+                    layer_alignment_loss = (1.0 - cosine).mean()
                 else:
-                    raise ValueError(f"Unsupported loss_type: {self.args.loss_type}")
+                    raise ValueError(f"Unsupported loss_type: {loss_type}")
+                alignment_loss = alignment_loss + layer_alignment_loss
+            loss = (1 - hybrid_alpha) * loss + hybrid_alpha * alignment_loss
```
Comment (mrsndmn) [inline src/compression_horizon/train/trainer.py]: обычно лоссы выглядят просто как взвешенная сумма, я бы рекомендовал сделать его в таком виде:
```python
            loss = loss + hybrid_alpha * alignment_loss
```

File: tests/bar.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import torch
```
[the remark this answers is not in the document]
Comment (M1croZavr) [reply tests/bar.py]: Да
